spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -52,134 +52,45 @@ http://start.spring.io/[Spring Initializr] and creating a new project. Doing so
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
-	<modelVersion>4.0.0</modelVersion>
-
-	<groupId>com.example</groupId>
-	<artifactId>myproject</artifactId>
-	<packaging>jar</packaging>
-	<version>0.0.1-SNAPSHOT</version>
-
-	<parent>
-		<groupId>org.springframework.boot</groupId>
-		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>1.5.2.RELEASE</version>
-	</parent>
-
-	<properties>
-		<start-class>com.example.SampleTask</start-class>
-	</properties>
-
-	<dependencies>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter</artifactId>
-		</dependency>
-	</dependencies>
-
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.springframework.boot</groupId>
-				<artifactId>spring-boot-maven-plugin</artifactId>
-			</plugin>
-		</plugins>
-	</build>
-</project>
-----
-
-Creating a `pom.xml` file with the preceding content should give you a working build. You
-can test it by running `mvn package` (for now, you can ignore the "jar will be empty - no
-content was marked for inclusion!" warning ).
-
-NOTE: At this point, you could import the project into an IDE (most modern Java IDE's
-include built-in support for Maven). For simplicity, we continue to use a plain text
-editor for this example.
+To do so:
 
-[[getting-started-adding-classpath-dependencies]]
-=== Adding classpath dependencies
-
-A Spring Cloud Task is made up of a Spring Boot application that is expected to end. In
-the `pom.xml` file we showed earlier, we created the shell of a Spring Boot application by
-setting our parent to use the `spring-boot-starter-parent`.
-
-Spring Boot provides a number of additional "`Starter POMs`". Some of them are appropriate
-for use within tasks (`spring-boot-starter-batch`, `spring-boot-starter-jdbc`, and
-others), and some may not be ('spring-boot-starter-web` is probably not going to be used
-in a task). The best indicator of which starter makes sense is whether the resulting
-application should end. Batch-based applications typically end. Conversely, the
-`spring-boot-starter-web` dependency bootstraps a servlet container, which better suits
-applications that continue.
-
-For this example, we need only to add a single additional dependency -- the one for
-Spring Cloud Task itself:
-
-[source,xml]
-		<dependency>
-			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-starter-task</artifactId>
-			<version>1.2.3.RELEASE</version>
-		</dependency>
+. Visit the link:https://start.spring.io/[Spring Initialzr] site.
+.. Create a new Maven project with a *Group* name of `io.spring.demo` and an *Artifact* name of `helloworld`.
+.. In the Dependencies text box, type `task` to select the `Cloud Task` dependency.
+.. In the Dependencies text box, type `jdbc` then select the `JDBC` dependency.
```

Comment:
Make "then select" read "and then select" ("then" is not a conjunction, so we have to have "and" there, too).